app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/model/DummyCategory.kt

```diff
@@ -11,7 +11,7 @@ import kotlinx.parcelize.Parcelize
  * @param representativePicture a categorized picture, can be null
  */
 @Parcelize
-data class DummyCategory(override val id: String, override val name: String, override val representativePicture: CategorizedPicture?
+class DummyCategory(override val id: String, override val name: String, override var representativePicture: CategorizedPicture?
```

Comment:
I don't know about this design decision. If Categories get parceled, are you going to track every instance? And in the FireStore implementation it's even worse. Better to keep immutability imho

Reply:
Agreed. But making it a data class was breaking my tests, I'll try to figure out why tomorrow and change it back if so.